### DISCUSSION TITLE
Generating Offline Tile Map service (TMS) and correspoding XML file used in osg 3D engine: SRTM Elevation data in GeoTiff format

### DISCUSSION THREAD

**Discussion by shramir** [pelicanmapping/osgearth#1990] [Support Q&A]
Hi,
I have searched for this question **over one year**!! 
I need to generate an offline **large scale**(south america) TMS and corresponding XML file in WGS84 projection to be connected and shown in OpenSceneGraph 3D engine. The elevation file format should be GeoTiff and extracted from the SRTM database.

**Comment by XenonofArcticus** [comment]
You probably need to supply more info.

What have you already tried and how was it insufficient?
